nano/templates/newscaster.tmpl

```diff
@@ -139,11 +139,14 @@ Property of Nanotrasen</font><br>
 		{{else}}
 			{{for data.messages}}
 				<div class='statusDisplay'>
-					<b>{{:value.title}}</b><br>
-					{{:value.body}}<br>
+					<p align="center"> 
+					<font size="5">
+					<b>{{:value.title}}</b></font><br>
 					{{if value.img}}
-						<img src='data:image/png;base64,{{:value.img}}' width=180><br>
-					{{/if}}
+					<img src='tmp_photo{{:value.message_number}}.png' width=350><br>
+					{{/if}}		
+					<p align="justify"> 					
```

Comment:
In testing the closing P's didn't seem to be required at all, but I have added them along with the indentations you suggested!

Reply:
without them lines 151-152 would also be using the "align = justify" and anything else added to the div below it

it tells it when to end, see #9795, PAIs were messed up just because it was missing a closing tag, you should always have closing tags on things even though it may work without them

Reply:
Roger that!

How does it look as is?

Reply:
the newscaster.tmpl looks good to me

Reply:
Huzzah! Should be fit for a merge I reckon!